androidApp/src/main/res/values/strings.xml

```diff
@@ -22,8 +20,8 @@
     <string name="more_title">MBTA Go</string>
     <string name="nearby_transit_link">Nearby</string>
     <string name="no_service">No Service</string>
-    <string name="no_stops_nearby">Your current location is outside of our search area.</string>
-    <string name="no_stops_nearby_title">No nearby MBTA stops</string>
+    <string name="no_stops_nearby">You’re outside the MBTA service area.</string>
+    <string name="no_stops_nearby_title">No nearby stops</string>
```

Comment:
This text changed in iOS, so to get it to match in Android (and thereby not fail the build due to `MissingTranslation`) I've gone ahead and updated it here.